core/src/test/java/org/infinispan/dataconversion/DataConversionTest.java

```diff
@@ -229,6 +234,7 @@ public void call() throws InterruptedException {
       });
    }
 
+   @SuppressWarnings("unused")
    private class TestInterceptor extends BaseCustomAsyncInterceptor {
```

Comment:
This inner class could be ``private static`` too

Reply:
changed!